lib/src/main/java/uk/gov/di/ipv/cri/passport/library/error/OAuthErrorResponse.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.di.ipv.cri.passport.library.error;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OAuthErrorResponse {
+    public static final String SERVER_ERROR_CODE = "server_error";
+    private final String error;
+    private final String error_description;
+
+    public OAuthErrorResponse(
```

Comment:
I don't think we need this new class. There is some from the nimbus library that we could use.

There is the ErrorObject class which we can instantiate our own and pass the error codes and descriptions into.